src/ext/java/org/opentripplanner/ext/fares/FaresConfiguration.java

```diff
@@ -99,6 +100,7 @@ private static FareServiceFactory createFactory(String type) {
         "highestFareInFreeTransferWindow" -> new HighestFareInFreeTransferWindowFareServiceFactory();
       case "hsl" -> new HSLFareServiceFactory();
       case "atlanta" -> new AtlantaFareServiceFactory();
+      case "orca" -> new OrcaFareFactory();
```

Comment:
Can you please document this in https://github.com/opentripplanner/OpenTripPlanner/blob/c1d4d8fdf7ecd325776188d97241a145224eb496/docs/sandbox/Fares.md ?